Gemfile

```diff
@@ -54,7 +54,7 @@ group :test do
   gem 'database_cleaner'
   gem 'rspec-rails'
   gem 'gherkin'
-  gem 'factory_girl_rails'
+  gem 'factory_bot_rails'
   gem 'faker'
   gem 'fakeweb'
   gem 'capybara'
```

Comment:
A lot of muddy work regarding cucumber, capybara, and related gems here... there's actually a downgrade to one of them in this change set. Due to major gherkin changes along with capybara matchers and other "stuff", someone more knowledgeable than I would need to more or less audit and/or rewrite the feature tests. For now, this works, minus what SHOULD be a safe "error" where minitest is invoked accidentally and complains about an unknown `--format` flag.